tests/smoke/test_cli.py

```diff
@@ -41,3 +42,11 @@ def test_cli_reg_metrics_runs_with_output(capfd, tmpdir):
     os.system(f'reg_metrics data/test_data/ --tiff_list input_1500.tif --do_registration 2 --save_path0 {tmpdir}')
     captured = capfd.readouterr()
     assert captured.out
+
+
+def test_cli_suite2p_gui_runs_when_is_called_locally():
+    s2p = subprocess.Popen(['python', '-m', 'suite2p'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    sleep(10)
+    s2p.terminate()
+    output, err = s2p.communicate(b'\n')
+    assert "Error" not in err.decode('ascii')
```

Comment:
I'm not sure I understand the purpose of these two lines.  What do they do?

Reply:
The first line gives access to the output and error logs of the process. The output/error is in byte format though so the second line converts it to str.

The lines are making sure that no error is present during that time the `suite2p` command runs